Dockerfile

```diff
@@ -96,6 +96,22 @@ RUN set -x && \
     cp /go/LICENSE /usr/local/share/package-licenses/go.LICENSE && \
     cp LICENSE /usr/local/share/package-licenses/PMEM-CSI.LICENSE
 
+# Some of the licenses might require us to distribute source code.
+# We cannot just point to the upstream repos because those might
+# disappear. We could host a copy at a location under our control,
+# but keeping that in sync with the published container images
+# would be tricky. So what we do instead is copy the (small!)
+# source code which has this requirement into the image.
+RUN set -x && \
```

Comment:
what was the reason to not have it in a separate script, like `hack/copy-module-licenses.sh`?

Reply:
No particular reason. It seemed simple enough to do it directly in the Dockerfile.